client/src/charts/nivo/nivo_line.js

```diff
@@ -93,19 +93,32 @@ export class NivoResponsiveLine extends React.Component {
       .compact()
       .groupBy('label')
       .map( _.spread(_.merge) )
-      .map( row=> ({
-        display_values: _.mapValues(
-          row,
-          (values, key) => key === 'label' ? values : get_formatter(is_money,text_formatter,true,true)(values)
-        ),
-        sort_values: row,
-        search_values: {label: row.label},
+      .map( row => ({
+        ..._.mapValues(row),
       }))
       .value();
-    const ordered_column_keys = _.concat( ['label'], table_ordered_column_keys || _.map(data,'id') );
-    const column_names = _.concat( [table_first_column_name || graph_text_maker("label")], table_ordered_column_keys || _.map(data,'id') );
+    const column_configs = {
+      label: {
+        index: 0,
+        header: table_first_column_name || graph_text_maker("label"),
+        is_sortable: true,
+        is_searchable: true,
+      },
+      ..._.chain(table_ordered_column_keys || _.map(data,'id'))
+        .map( (col, idx) => [col, {
+          index: idx + 1,
+          header: col,
+          is_sortable: true,
+          formatter: (value) => value ? get_formatter(is_money,text_formatter,true,true)(value) : "",
```

Comment:
Just noting that this will make values of zero empties. This is a tricky subject because empties get turned in to zeros when the datasets are read in... but I think we tend to just display them as zeros always because empties are more confusing/zero is usually technically correct still?

Also, the bar chart should have the same check added, saw some tables for bar charts full of stuff like "$NaN", :rofl:

Reply:
Were you seeing $NaN for bar charts in this branch? I'm seeing values of zero, displayed as $0 for bar charts, and empties with line chart (I believe because of this check). Which should I be going for?

Reply:
I think we've always gone with zeros, double check how the current master versions do it. I'd try and find the NaNs I was seeing, but I'm getting problems on the dev links. Not your fault, CORS rules are causing a problem; Google changed their default permissions a while back but I swore I fixed it... Maybe rebase from master and force push up? That might fix the dev links.

Reply:
Okay!

Reply:
In master, bar is displayed as 0, line is displayed as empty (which is exactly the same case as this branch)